unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowHUDController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using DCL;
+using DCL.Interface;
+using System.Collections;
+using UnityEngine;
+
+public class PrivateChatWindowHUDController : IHUD
+{
+    private ChatHUDController chatHudController;
+    public PrivateChatWindowHUDView view;
+
+    private IChatController chatController;
+
+    internal bool resetInputFieldOnSubmit = true;
+
+    public void Initialize(IChatController chatController, IMouseCatcher mouseCatcher)
```

Comment:
Here, we lack some kind of "user ownership" of the window. We should define some interface that'd allow the FriendsHUD/TaskbarHUD to re-populate the window properly.